js_modules/dagit/packages/core/src/instigation/TickHistory.tsx

```diff
@@ -253,6 +265,11 @@ export const TicksTable = ({
                     {tick.logKey ? <a onClick={() => setLogTick(tick)}>View logs</a> : <>&mdash;</>}
                   </td>
                 ) : null}
+                <td>
+                  {tick.dynamicPartitionsRequestResults.length ? (
+                    <DynamicPartitionRequestsCell requests={tick.dynamicPartitionsRequestResults} />
+                  ) : null}
```

Comment:
Show "None" or something like that, instead of the empty cell?

You can also probably consolidate this ternary away, and always render the `DynamicPartitionRequestsCell`, since it will render to null in the empty array case anyway.